### PR TITLE
add canvas to react

### PR DESCRIPTION
**Pull request by DrabykMykola**

### CODE REVIEW

**Review by tfedorov — APPROVED** [review]

File: frontend/src/components/pages/Canvas/Canvas.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        Background = new component(window.innerWidth, window.innerHeight, "/pictures/images-for-canvas/map/map.jpg", 0, 0, "image");
+
+        Marker = new component(125, 200, "/pictures/images-for-canvas/marker/marker.svg", 780, 350, "image");
+
+        // GamePiece = new component(10, 20, "images/cursor/cursor_default.jpg", 0,0, "image");
+        GameArea.start();
+    }
+    startGame();
+})
+console.log(window.location.pathname)
```
Comment (SeriyBg) [inline frontend/src/components/pages/Canvas/Canvas.js:21]: Do we need this?

File: frontend/src/components/pages/Canvas/Canvas.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+var GameArea = {
+    canvas : document.createElement("canvas"),
+    start : function() {
+        this.canvas.width = window.innerWidth;
+        this.canvas.height = window.innerHeight;
+        // this.canvas.style.cursor = "none";
```
Comment (SeriyBg) [inline frontend/src/components/pages/Canvas/Canvas.js:29]: Commented code is not good to have in the production code 😃

File: frontend/src/components/pages/Canvas/Canvas.js (hunk 1 above)
Comment (DrabykMykola) [reply frontend/src/components/pages/Canvas/Canvas.js:21]: ok

File: frontend/src/components/pages/Canvas/Canvas.js (hunk 2 above)
Comment (DrabykMykola) [reply frontend/src/components/pages/Canvas/Canvas.js:29]: ok

**Comment by DrabykMykola** [comment]
need to rework

**Pull request closed without merging**